src/controllers/commands/CommandController.cpp

```diff
@@ -889,9 +895,27 @@ void CommandController::initialize(Settings &, Paths &paths)
                 return "";
             }
 
-            channel->addMessage(makeSystemMessage(
-                QString("Chatter count: %1")
-                    .arg(localizeNumbers(twitchChannel->chatterCount()))));
+            auto addChatterCountMessage = [channel, twitchChannel]() -> void {
+                channel->addMessage(makeSystemMessage(
+                    QString("Chatter count: %1")
+                        .arg(localizeNumbers(twitchChannel->chatterCount()))));
+            };
+
+            // Fetch from chatter cache when user does not have mod rights
+            if (!twitchChannel->hasModRights()) {
+                addChatterCountMessage();
```

Comment:
Given my previous comment above, I would prefer to return an error message for non-mods rather than providing inaccurate information.

Reply:
Added an error message.